src/modules/px4iofirmware/mixer.cpp

```diff
@@ -287,19 +295,29 @@ mixer_tick(void)
 		/* mix */
 		mixed = mixer_mix_threadsafe(&outputs[0], &r_mixer_limits);
 
-		/* the pwm limit call takes care of out of band errors */
-		pwm_limit_calc(should_arm, should_arm_nothrottle, mixed, r_setup_pwm_reverse, r_page_servo_disarmed,
-			       r_page_servo_control_min, r_page_servo_control_max, outputs, r_page_servos, &pwm_limit);
+		/* if mixer_mix_threadsafe returns zero, it did nothing */
+		if (mixed != 0) {
```

Comment:
Good point, I didn't see that. Let's check:

Before a new mixer is set, the flag is reset:
https://github.com/PX4/Firmware/blob/e7db0ed098bcc030042aba0667d09da67c130ffe/src/modules/px4iofirmware/mixer.cpp#L483

If a mixer is set/sent correctly the flag is set:
https://github.com/PX4/Firmware/blob/27abc4fe6b07a214125af7ea7f595f2d0bb15bb8/src/drivers/px4io/px4io.cpp#L2304

If overflow occurs in setting the mixer, the flag is also reset:
https://github.com/PX4/Firmware/blob/e7db0ed098bcc030042aba0667d09da67c130ffe/src/modules/px4iofirmware/mixer.cpp#L514

I don't see any other place where this flag is set, so it looks like there is no chance this will be triggered in flight, unless you send a new mixer, and that's probably forbidden as well.